rules-tests/TypeDeclaration/Rector/ClassMethod/StrictStringParamConcatRector/Fixture/skip_inner_closure.php.inc

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\StrictStringParamConcatRector\Fixture;
+
+final class SkipInnerClosure
+{
+    public function resolve($item)
+    {
+        $result = $item * 100;
+
+        function ($item) {
+            return $item . ' world';
+        };
```

Comment:
inner closure can have result `function ($item): string {` by add `\PhpParser\Node\Expr\Closure::class` to `getNodeTypes()` method

```diff
-        return [ClassMethod::class, Function_::class];
+        return [ClassMethod::class, Function_::class, \PhpParser\Node\Expr\Closure::class];
```

Reply:
This is skipped on purpose, to avoid higher complexity due to inner closure inheritances. These strict rules should only handle method + function, nothing else.